### PR TITLE
Add Int.ListFormat Demo

### PR DESCRIPTION
**Pull request by romulocintra**
Demo to be embed  in [ListFormat Page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ListFormat)

### CODE REVIEW

**Comment by romulocintra** [comment]
cc @wbamberg

File: live-examples/js-examples/intl/intl-listformat.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">const list = ['Motorcycle', 'Bus', 'Car'];
+
+console.log(new Intl.ListFormat('en-GB', { style: 'long', type: 'conjunction' }).format(list));
+// > Motorcycle, Bus and Car
```
Comment (wbamberg) [inline live-examples/js-examples/intl/intl-listformat.html]: This should be `// expected output: "Motorcycle, Bus and Car"`

File: live-examples/js-examples/intl/intl-listformat.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+console.log(new Intl.ListFormat('en-GB', { style: 'short', type: 'disjunction' }).format(list));
+// > Motorcycle, Bus or Car
```
Comment (wbamberg) [inline live-examples/js-examples/intl/intl-listformat.html]: `// expected output: "Motorcycle, Bus and Car"`

File: live-examples/js-examples/intl/intl-listformat.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+console.log(new Intl.ListFormat('en-GB', { style: 'narrow', type: 'unit' }).format(list));
+// > Motorcycle Bus Car
```
Comment (wbamberg) [inline live-examples/js-examples/intl/intl-listformat.html]: `// expected output: "Motorcycle Bus Car"`

File: live-examples/js-examples/intl/intl-listformat.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">const list = ['Motorcycle', 'Bus', 'Car'];
```
Comment (wbamberg) [inline live-examples/js-examples/intl/intl-listformat.html]: I'd prefer, say, `vehicles` rather than `list` as the variable name.

**Pull request merged**